src/lib/chips/chips.scss

```diff
@@ -19,6 +19,16 @@ $mat-chip-input-margin: 3px;
   align-items: baseline;
 }
 
+.mat-form-field-type-mat-chip-list {
```

Comment:
should not be necessary, the user docs are apparently out of date, chips should be done like this now, which resolves the wrapping issues: http://plnkr.co/edit/kc3bxAZRgHS0JbGl3qSX?p=preview

Reply:
Done. I'll make a PR to update the docs too :)